MWWebPlugin/MWWebPlugin/WebStep/MWWebViewController.swift

```diff
@@ -273,6 +276,13 @@ extension MWWebViewController: WKNavigationDelegate {
             await self.show(error)
         }
     }
+    
+    public func webView(_ webView: WKWebView, didFailProvisionalNavigation navigation: WKNavigation!, withError error: Error) {
+        Task {
```

Comment:
@FWJonathan Instead of replicating the exact same logic here, it is better to create a method that both delegate methods can call to show errors.